ngraph/core/include/ngraph/op/constant.hpp

```diff
@@ -483,6 +344,15 @@ namespace ngraph
                                    [](IN_T c) { return static_cast<OUT_T>(c); });
                 }
 
+                template <element::Type_t Type,
+                          typename T,
+                          typename DataStorageType = fundamental_type_for<Type>>
+                void fill_data(T value)
```

Comment:
Looks like an extra copy of the data that is already copied inside the Constant's constructor. Consider `T&` or `T&&`.

Reply:
done.